### PR TITLE
Add pretix-presale-saml docs

### PR DESCRIPTION
**Pull request by pc-coholic**

### CODE REVIEW

File: doc/plugins/presale-saml.rst [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+``Admin Mode`` button in the top right hand corner.
+
+You should now see a new menu-item titled ``SAML`` appear.
+
+Organizer Level
+^^^^^^^^^^^^^^^
+
+Navigate to the organizer settings in the pretix backend. In the navigation
+bar, you will find a menu-item titled ``SAML`` if your user has the ``Can
+change organizer settings`` permission.
```
Comment (raphaelm) [inline doc/plugins/presale-saml.rst:73]: I don't feel that this is the correct markup for non-code, I'd either use italics or just quotation marks

**Comment by raphaelm** [comment]
I think after you made the spellcheck happy (maybe by adding a [file-specific wordlist to the top of the new file](https://sphinxcontrib-spelling.readthedocs.io/en/latest/customize.html#private-dictionaries)), we can just merge this

File: doc/plugins/presale-saml.rst [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+    to buy a ticket, you can provide them a small blurb here. Markdown is supported.
+
+``Attribute RegEx``
+    By default, any successful authentication with the IdP will allow the user to
+    proceed with their purchase. Should the allowed audience needed to be restricted
+    further, a set of regular Expressions can be used to do this.
+
+    An Attribute RegEx of ``{}`` will allow any authenticated user to pass.
+
+    A RegEx of ``{ "affiliation": "(employee@pretix.eu|staff@pretix.eu)" }`` will
```
Comment (raphaelm) [inline doc/plugins/presale-saml.rst:210]: The example should probably be this?

```suggestion
    A RegEx of ``{ "affiliation": "^(employee@pretix.eu|staff@pretix.eu)$" }`` will
```

Comment (pc-coholic) [reply doc/plugins/presale-saml.rst:210]: It actually also works without the ^ and $... Just tried it out again...

Comment (raphaelm) [reply doc/plugins/presale-saml.rst:210]: Did you just try that someone with ``staff@pretix.eu`` gets in correctly or did you also try if someone affiliated with ``staff@pretix.eu.myevildomain.com`` is actually blocked?

Comment (pc-coholic) [reply doc/plugins/presale-saml.rst:210]: Oh, I see your point now... I did indeed only test the first thing...

Adding this here *and* updating the plugin itself, too.

**Pull request merged**